wcivf/apps/elections/tests/test_election_views.py

```diff
@@ -47,29 +46,46 @@ def test_election_detail_view(self):
         self.assertTemplateUsed(response, "elections/election_view.html")
         self.assertContains(response, self.election.nice_election_name)
 
-    @pytest.mark.freeze_time("2017-03-23")
-    def test_election_detail_day_of_election(self):
```

Comment:
Has this tet been deleted accidentally or was there a problem with it?

Reply:
Accidentally! During a merge conflict. I'll add back.